example/lib/utilities/helper.dart

```diff
@@ -0,0 +1,14 @@
+class Helper {
+  static String formatDateTime(int time) {
```

Comment:
actually move this back to the chat summary page. this is specific to that page not a common formatter, I thought this was for showing time on messages